File: Pacman-Game-OOP-master 2/EASendMail/Samples_ASPNetMvc/VB/WebProject1/WebProject1/Models/DbRecipientResult.vb
Imports System.ComponentModel.DataAnnotations

Public Class DbRecipientResult
    <Key, ScaffoldColumn(False), Display(Name:="Result Id")>
    Public Property ResultId As Integer
    <Required, Display(Name:="Task Id")>
    Public Property TaskId As Integer
    <MaxLength(250)>
    Public Property Recipient As String
    Public Property Status As String
    <Display(Name:="Succeeded")>
    Public Property IsSucceeded As Boolean
    <Display(Name:="Date Time")>
    Public Property CreationTime As DateTime
End Class

File: Pacman-Game-OOP-master 2/EASendMail/Samples_ASPNetMvc/C#/WebProject1/WebProject1/Models/DbRecipientResult.cs
using System;
using System.ComponentModel.DataAnnotations;
namespace WebProject1.Models
{
    public class DbRecipientResult
    {
        [Key, ScaffoldColumn(false), Display(Name ="Result Id")]
        public int ResultId { get; set; }

        [Required, Display(Name ="Task Id")]
        public int TaskId { get; set; }

        [MaxLength(250)]
        public string Recipient { get; set; }

        public string Status { get; set; }

        [Display(Name = "Succeeded")]
        public bool IsSucceeded { get; set; }

        [Display(Name = "Date Time")]
        public DateTime CreationTime { get; set; }
    }
}